articles/azure/azs-how-create-service-fabric-portal.md

```diff
@@ -158,11 +160,23 @@ You also need the thumbprint of the certificate(s). For information about how to
 
    - **Repair Manager** - Select **Yes** to use Repair Manager to enable patch orchestration on node types with a durability of bronze, which helps keep your VMs up to date.
 
-   - **Service Fabric deployment package URL** - Specify the URL to download the Service Fabric deployment package from. For off-line scenarios, download the service fabric package from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.
+   - **Diagnostic Data Age In Days** - Specify the days to keep the Service Fabric diagnostic log in the diagnostics store.
+
+   - **Service Fabric deployment package URL** - Specify the URL to download the Service Fabric deployment package from. For disconnected scenarios, download the service fabric package from the URL specified and upload it to a blob and enable anonymous access and specify the URL here.

```

Comment:
Lots of "ands" in this paragraph and the next one! Suggest:
...download the service fabric package from the URL specified and upload it to a blob, then enable anonymous access and specify the URL here.

Reply:
Fixed in commit 1962cd6